### PR TITLE
Update vsb and vsr api refs

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**

### CODE REVIEW

File: docs/vsb_api_ref.md [hunk 1]
```diff
@@ -2,29 +2,31 @@
...
-| ResticRepository      | string                                             | ResticRepository is the location in which the snapshot will be stored.        |
-| Phase      | VolumeSnapshotBackupPhase                                       | Phase is the VolumeSnapshotBackup phase status.           |
-
+| Property             | Type                      | Description                                                                 |
+|----------------------|---------------------------|-----------------------------------------------------------------------------|
+| Completed     | bool                      | Completed is whether or not VolumeSnapshotBackup has completed reconciling. |
+| SourcePVCData      | PVCData                   | SourcePVCData is a reference to the source PVC.                             |
+| ResticRepository      | string                    | ResticRepository is the location in which the snapshot will be stored.      |
+| Phase      | VolumeSnapshotBackupPhase | Phase is the VolumeSnapshotBackup phase status.                             |
```
Comment (eemcmullan) [inline docs/vsb_api_ref.md:19]: `Phase` is what we update with VolSync CR state as they are running, right?

File: docs/vsb_api_ref.md [hunk 2]
```diff
@@ -2,29 +2,31 @@
...
+| Conditions      | []metav1.Condition        | Include references to the volsync CRs and their state as they are running   |
```
Comment (eemcmullan) [inline docs/vsb_api_ref.md:20]: See above comment

File: docs/vsr_api_ref.md [hunk 3]
```diff
@@ -13,8 +13,9 @@
 
 | Property             | Type                        | Description                                 |
 |----------------------|-------------------------------------------------|------------------------------------------------------|
-| Completed     | bool                                                    | Completed is whether or not VolumeSnapshotRestore has completed reconciling.    |
+| Phase     | VolumeSnapshotRestorePhase                                                    | volumesnapshot restore phase status    |
 | SnapshotHandle     | string                                             | SnapshotHandle is the snaphandle from the volumeSnapshotContent created by VolSync.      |
+| Conditions     | []metav1.Condition                                                 | Include references to the volsync CRs and their state as they are running     |
```
Comment (eemcmullan) [inline docs/vsr_api_ref.md:18]: See comment about `phase` for VSB above

File: docs/vsb_api_ref.md (hunk 1 above)
Comment (shubham-pampattiwar) [reply docs/vsb_api_ref.md:19]: Not exactly, Volsync CR states would be a subset of datamover phases, right ?

**Review by eemcmullan — APPROVED** [review]

**Pull request merged**